tests/test_difficulty.py

```
@@ -0,0 +1,63 @@
+        chain2.avg_block_time = 1000
+        chain2.current_difficulty = 1
+        chain2.chain[0].difficulty = 1
+
+        # Chain 2 mines a fast block, difficulty goes to 2
+        block1 = Block(1, chain2.last_block.hash, [], timestamp=chain2.last_block.timestamp + 1, difficulty=chain2.current_difficulty, state_root=chain2.state.state_root())
+        mine_block(block1)
+        chain2.add_block(block1)
+        self.assertEqual(chain2.current_difficulty, 2)
+        
+        # Reorg chain1 to chain2
+        success, orphans = chain1.resolve_conflicts(chain2.chain)
```

Ruff (0.15.20)

[warning] Unpacked variable `orphans` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)